blocks/identity-block/components/Button/index.jsx

```diff
@@ -40,19 +42,41 @@ const StyledDynamicButton = styled.button.attrs((props) => ({
           background-color: #ffffff;
           border-color: #ffffff;
           color: ${getThemeStyle(arcSite)['primary-color']};
+
+          &:hover {
+            color: ${getThemeStyle(arcSite)['primary-color']};
```

Comment:
You could use `color: inherit` here

Reply:
yeah I was hoping the scoping would be easier as well. seems to be inheriting from another styled component's a:hover though: 

with color: inherit under hover

<img width="110" alt="Screen Shot 2021-08-31 at 10 00 28" src="https://user-images.githubusercontent.com/5950956/131527469-92f69559-a031-448b-855c-644b959989fa.png">

<img width="946" alt="Screen Shot 2021-08-31 at 10 07 29" src="https://user-images.githubusercontent.com/5950956/131527875-7bbd855b-647b-4bf2-835c-a7b104110283.png">


without inherit: 

![Screen Shot 2021-08-31 at 10 05 18](https://user-images.githubusercontent.com/5950956/131527531-e1d5e443-7fa9-41f9-9f4c-1673d696d994.png)
<img width="113" alt="Screen Shot 2021-08-31 at 10 03 29" src="https://user-images.githubusercontent.com/5950956/131527551-c57ab7f3-8749-4585-b61b-78faadce3d0e.png">